packages/story-editor/src/components/library/common/searchInput.js

```diff
@@ -58,50 +49,32 @@ function WrappedSearchInput({
   placeholder,
   onSearch,
   disabled = false,
-  incremental = false,
-  delayMs = 500,
 }) {
   // Local state so that we can debounce triggering searches.
   const [localValue, setLocalValue] = useState(initialValue);
 
-  // Effectively performs a search, triggered at most every `delayMs`.
-  const changeSearchTermDebounced = useDebouncedCallback(() => {
-    onSearch(localValue);
-  }, delayMs);
-
-  const submitValue = useCallback(
-    (value) => {
-      if (incremental && delayMs) {
-        changeSearchTermDebounced();
-      } else {
-        onSearch(value);
-      }
-    },
-    [changeSearchTermDebounced, onSearch, delayMs, incremental]
-  );
-
   const onChange = useCallback(
```

Comment:
@swissspidy 

Should we remove the onChange as well and let onClear or "" + Enter --- clear the search ?

If yes I should be able to remove the `localValue` as well.